webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -158,24 +143,24 @@ const WorkspaceDetails: React.FC = () => {
   );
 
   const tabItems: TabsProps['items'] = useMemo(() => {
-    if (!workspace) {
-      return [];
-    }
-
-    const items: TabsProps['items'] = [
-      {
-        children: <WorkspaceProjects id={id} pageRef={pageRef} workspace={workspace} />,
-        key: WorkspaceDetailsTab.Projects,
-        label: 'Projects',
-      },
-      {
-        children: <TaskList workspace={workspace} />,
-        key: WorkspaceDetailsTab.Tasks,
-        label: 'Tasks',
-      },
-    ];
+    const items: TabsProps['items'] = Loadable.match(workspace, {
+      Loaded: (w: Workspace) => [
+        {
+          children: <WorkspaceProjects id={id} pageRef={pageRef} workspace={w} />,
+          key: WorkspaceDetailsTab.Projects,
+          label: 'Projects',
+        },
+        {
+          children: <TaskList workspace={w} />,
+          key: WorkspaceDetailsTab.Tasks,
+          label: 'Tasks',
+        },
+      ],
+      NotLoaded: () => [],
+    });
 
-    if (rbacEnabled) {
+    const workspaceLoaded = Loadable.getOrElse(undefined, workspace);
```

Comment:
**nit:** You can keep this as is if you want. I have the convention in other places of:
```
const loadableItem: Loadable(Item)
const item: Item | undefined;
```

Reply:
also might be better to put this line directly underneath the other variable?

Reply:
yes I'll move this to the top under the observable code